docs/.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.2.1
+3.1.2
```

Comment:
Did you change it deliberately, and if so, why? I think it was changed as part of the PR [#149 ](url)

Reply:
I removed this to maintain it same as main branch.